test/e2e/rules_test.go

```diff
@@ -12,6 +12,65 @@ import (
 	"github.com/efficientgo/tools/core/pkg/testutil"
 )
 
+const recordingRuleYamlTpl = `
```

Comment:
why not keep them in `test/e2e/configs.go`?

Reply:
I felt like it is easier to find these in the same file as rule tests, since it's strictly not a test service configuration, but rather test data.